### PR TITLE
Change 10461 limit bulk action load

### LINKED ISSUE #10461: Limit bulk action load for "Edit" bulk actions [5]
### Problem Description

When performing the Delete Bulk Action for a large number of Travel Entries (18.240 Travel Entries), takes such a long time that the server is an infinite loop.
I waited more than 40 minutes and nothing happened, I had to restart the server.
Unfortunately, the logs don't show anything.

<details>
<summary>Details</summary>

#### Steps to Reproduce

1. Open the Travel Entries Directory and make sure you have a huge number of Travel Entries 
(I attach a file with 18.240 Travel Entries: [deaimport (18240).csv](https://github.com/hzi-braunschweig/SORMAS-Project/files/9655050/deaimport.18240.csv)
2. Enter the Bulk Edit Mode and select all;
3. Click on Bulk Actions and then click on 'Delete'

#### Expected Behavior
The deletion should take a reasonable amount of time.

#### System Details

* Device: Windows 10
* SORMAS version: Sormas 1.76.0-SNAPSHOT
* Android version/Browser: Chrome
* Server URL: local environment
* User Role: admin + national user

</details>

### Proposed Change

- Still allow to select as much items as currently possible in the UI, but only process up to 500 entries or 60s (whatever comes first).
- Reload the UI with the leftovers (keep filtering, reselect all entries if possible), show a message how many items were processed in what amount of time and tell the user to continue if he likes to.
- Text message if max entries have been reached: ```The maximum amount of entries that can be processed at the same time have been reached (<maxEntryCount>). There are still (<entryCountLeft>) entries left to process. Do you want to continue the operation?```
- Text message if 60s have been reached: ```The bulk processing has been halted because the maximum amount of runtime has been reached (<maxRuntime>s). (<entryCountProcessed>) entries have been processed so far and there are still (<entryCountLeft>) entries left to process. Do you want to continue the operation?```

### Acceptance Criteria

- [ ] The user can easily continue by starting the bulk operation again.
- [ ] No bulk operation is noticed that still stalls the system.

### Implementation Details

- Create a new helper method that stops the bulk processing after 500 entries or 60s (whatever comes first) for all actions that currently process one entry per backend call. The user experience should be that the user gets an updated list with less entries so he can start the bulk action again to process the next entries.
- Debug logging that which bulk operation was interrupted, with amount of selected and processed entries, and how much time was spend.
- If we find any occurence where 500 entries or 60s still run into a deadlock, we should make the interrupt parameters flexible and lower the entry number where needed.

### Additional Information

- This does not change the architecture to directly speed up bulk operations (which is not easy and sometimes maybe not really possible), but limits the amount of data handled in the same transaction/roundtrip. The aim is to guarantee that operations finish within a defined timeframe instead of stalling to a complete stop.

### PR DESCRIPTION
**Pull request by MateStrysewske**
Fixes #10461

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: Please find the results at https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11442

File: sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java [hunk 1]
```diff
@@ -2366,10 +2367,11 @@ public int saveBulkContacts(
 				}
 
 				save(existingContactDto);
-				changedContacts++;
+				return true;
 			}
-		}
-		return changedContacts;
+
+			return false;
```
Comment (sergiupacurariu) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java:2373]: same as for cases and events

**Review by sergiupacurariu — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11442

**Review by sergiupacurariu — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11442

**Review by sergiupacurariu — APPROVED** [review]

**Pull request merged**